packages/autocomplete/src/containers/AutocompleteContainer.js

```diff
@@ -0,0 +1,563 @@
+     * @param {Any} renderProps.outOfBoundaries - Whether the current menu has escaped its boundary.
+     * @param {Any} renderProps.scheduleUpdate - Method to force an update within Popper.js
+     * @param {Any} renderProps.focusedKey - The currently focused key that was provided to `getItemProps()`
+     */
+    children: PropTypes.func,
+    /**
+     * Callback for when a menu item has been selected by keyboard or mouse. Return true from this callback to keep the menu open.
+     * @param {String} selectedKey - The key of the selected item
+     */
+    onSelect: PropTypes.func,
+    /**
+     * Whether Popper.js should update based on DOM resize events
+     */
+    eventsEnabled: PropTypes.bool,
+    /**
+     * The root ID to use for descendants. A unique ID is created if none is provided.
+     */
+    id: PropTypes.string,
+    /**
+     * Controls visibility of menu
+     */
+    isOpen: PropTypes.bool,
+    /**
+     * The currently focused item
+     */
+    focusedKey: PropTypes.string,
+    /**
+     * The currently focused tag
+     */
+    tagFocusedKey: PropTypes.string,
+    /**
+     * Returns all controlled state. Used for controlling usage.
+     */
+    onStateChange: PropTypes.func,
+    /**
+     * These placements differ from the default naming of Popper.JS placements to help
+     * assist with RTL layouts.
+     */
+    placement: PropTypes.oneOf([
+      'auto',
+      'top',
+      'top-start',
+      'top-end',
+      'end',
+      'end-top',
+      'end-bottom',
+      'bottom',
+      'bottom-start',
+      'bottom-end',
+      'start',
+      'start-top',
+      'start-bottom'
+    ]),
+    /**
+     * Passes options to [Popper.JS Instance](https://github.com/FezVrasta/popper.js/blob/master/docs/_includes/popper-documentation.md#new-popperreference-popper-options)
+     */
+    popperModifiers: PropTypes.object,
+    /**
+     * Identical to children
+     */
+    render: PropTypes.func,
+    /**
+     * @param {Object} renderProps
+     * @param {Function} renderProps.getInputProps - Props to be spread onto the input element
+     * @param {Function} renderProps.getTriggerProps - Props to be spread onto the containing, trigger element
+     * @param {Function} renderProps.getTagProps - Props to be spread onto each tag element. `({ key })` is required
+     * @param {Function} renderProps.triggerRef - Callback for the ref of the element triggering the menu
+     * @param {Function} renderProps.inputRef - Callback for the ref of the input
+     * @param {Boolean} renderProps.isOpen - Whether the Menu is currently visible
+     * @param {Boolean} renderProps.tagFocusedKey - The key of the currently focused tag
+     */
+    trigger: PropTypes.func,
+    /**
+     * The z-index of the popper.js placement container
+     */
+    zIndex: PropTypes.number
+  };
+
+  static defaultProps = {
+    placement: 'bottom-start',
+    eventsEnabled: true,
+    zIndex: 1000
```

Comment:
Does it make sense to expose and pull this default prop from `MenuContainer`?

Reply:
Yes, but unfortunately `react-styleguidist` doesn't recognized imported propTypes and defaultProps. There was a PR added for this, but seems like something they don't want to support.